java/short_url_generator/README.md

```diff
@@ -0,0 +1,56 @@
+# 💻 Get Short URL
+
+A Java Cloud Function for generating a Short URL using [tinyurl](https://tinyurl.com/app) and [bitly](https://bitly.com/)
+
+_Example input:_
```

Comment:
Let's please have one more example input with provider `bitly` so others can see what exact name they can use for provider.